batch_inference/batch_inference.py

```diff
@@ -38,19 +38,22 @@
 
 def _read_dataset_as_dataframe(
     dataset: DatasetType,
+    feature_columns: Union[str, List[str]] = None,
     label_columns: Union[str, List[str]] = None,
     drop_columns: Union[str, List[str], int, List[int]] = None,
 ) -> Tuple[pd.DataFrame, List[str]]:
     """
     Parse the given dataset into a DataFrame and drop the columns accordingly. In addition, the label columns will be
     parsed and validated as well.
 
-    :param dataset:       The dataset to train the model on.
-                          Can be either a list of lists, dict, URI or a FeatureVector.
-    :param label_columns: The target label(s) of the column(s) in the dataset. for Regression or
-                          Classification tasks.
-    :param drop_columns:  ``str`` / ``int`` or a list of ``str`` / ``int`` that represent the column names / indices to
-                          drop.
+    :param dataset:         The dataset to train the model on.
```

Comment:
shouldn't it be "The dataset the model was trained on" ?

Reply:
Fixed, actually it should be a dataset that will be converted into a DataFrame (doesn't have to be something that is related to a training set at all).